app/views/recipe/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="container">
+  <h1><%= @recipe.name %></h1>
+  <div class="d-flex justify-content-between">
+    <p>Preparation Time: <%= @recipe.preparation_time %> hours</p>
+    <div class="form-check form-switch">
+      <%= form_for @recipe, url: recipe_path(@recipe), method: :patch do |f| %>
+        <%= hidden_field_tag :authenticity_token, form_authenticity_token %>
+        <%= f.check_box :public, class: "form-check-input custom-toggle", id: "publicToggle", onchange: "this.form.submit();" %>
+        <%= f.label :public, class: "form-check-label", for: "publicToggle" do %>
+          Public
+        <% end %>
+        <%= f.submit class: "btn btn-primary d-none", value: "Save" %>
+      <% end %>
+    </div>
+  </div>
+  <p>Cooking Time: <%= @recipe.cooking_time %> minutes</p>
+
+  <div class="col-md-6">
+     <%= link_to "Generate Shopping list", new_recipe_food_path(recipe_id: @recipe.id), class: 'btn btn-primary btn-sm' %>
+    <%= link_to "Add Ingredient", new_recipe_food_path(recipe_id: @recipe.id), class: 'btn btn-primary btn-sm' %>
```

Comment:
- [ ] As required for the project **Generate Shopping list** to show the list of food that is missing for all recipes of the logged-in user and count the total food items and total price of the missing food. Kindly ensure that when clicking on the **Generate Shopping list** button it executes its expected task. Right now, the all above buttons are doing same thing of only adding **Gradients**. Please fix that.